sql-plugin/src/main/scala/org/apache/spark/sql/hive/rapids/GpuHiveOverrides.scala

```diff
@@ -48,12 +48,9 @@ object GpuHiveOverrides {
       GpuOverrides.expr[HiveSimpleUDF](
         "Hive UDF, support requires the UDF to implement a RAPIDS-accelerated interface",
         ExprChecks.projectNotLambda(
-          TypeSig.commonCudfTypes + TypeSig.ARRAY.nested(TypeSig.commonCudfTypes),
           TypeSig.all,
```

Comment:
We have never tested with any user defined types and I don't know what they might look like. At a minimum I think we want to exclude those until we have some kind of tests for them.

Reply:
I updated the types to include all but UDT.